src/renderer/coremods/settings/pages/General.tsx

```diff
@@ -110,6 +114,21 @@ export const General = (): React.ReactElement => {
         {intl.string(t.REPLUGGED_SETTINGS_QUICKCSS_AUTO_APPLY)}
       </SwitchItem>
 
+      {
+        // TODO: i18n
+        DiscordNative.process.platform.includes("linux") && (
+          <SwitchItem
+            value={titlebarValue}
+            onChange={(value) => {
+              titlebarOnChange(value);
+              restartModal(false);
```

Comment:
```suggestion
              restartModal(true);
```
A full app restart is required to load the custom titlebar.